composer.json

```diff
@@ -17,16 +17,16 @@
 	],
 	"require": {
 		"php": ">=5.3.2",
-		"symfony/form": "~2.3",
-		"symfony/http-kernel": "~2.3",
-		"symfony/translation": "~2.3",
-		"symfony/yaml": "~2.3"
+		"symfony/form": "^2.6.4|~3.0",
+		"symfony/http-kernel": "^2.6.4|~3.0",
+		"symfony/translation": "^2.6.4|~3.0",
+		"symfony/yaml": "^2.6.4|~3.0"
```

Comment:
if you drop support for the 2.3 LTS, I would suggest making 2.7 LTS your base version (2.6 does not receive bug fixes anymore, and will stop receiving security patches in 2 months)